src/pages/api/auth/register.ts

```diff
@@ -0,0 +1,84 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import { PrismaClient } from "@prisma/client";
+import bcrypt from "bcrypt";
+import jwt, { Secret } from "jsonwebtoken";
+
+const prisma = new PrismaClient();
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  if (req.method !== "POST") {
+    return res.status(405).json({ message: "Method not allowed" });
+  }
+
+  try {
+    const { name, username, email, password /*, repeatPassword*/ } = req.body;
+
+    // ? Do we need to check for empty fields
+    // ? Client-side validation should be enough
+    // if (!name || !username || !email || !password || !repeatPassword) {
+    //  return res.status(400).json({ message: "All fields are required" });
+    // }
+
+    // if (password !== repeatPassword) {
+    //   return res.status(400).json({ message: "Passwords do not match" });
+    // }
```

Comment:
same thing

Reply:
Erledigt in [deab844](https://github.com/top-22/forum/pull/14/commits/deab8447c0f39ff65214be94f11b32a650669b48)